From e919ced3e4467ab2b588e608db705c4035b2123e Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Fri, 12 Oct 2018 11:54:51 +0800
Subject: [PATCH] arm64: rk1808_linux_defconfig: disable
 CONFIG_PHY_ROCKCHIP_EMMC

The rockchip-emmc-phy driver is used for rk3399, so disable it
for rk1808.

Change-Id: I1642ff07b8749a1487b5515f86dfb1161151409d
Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
---
 arch/arm64/configs/rk1808_linux_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/rk1808_linux_defconfig b/arch/arm64/configs/rk1808_linux_defconfig
index 4c08e6882a82..dec93058b005 100644
--- a/arch/arm64/configs/rk1808_linux_defconfig
+++ b/arch/arm64/configs/rk1808_linux_defconfig
@@ -275,7 +275,6 @@ CONFIG_PWM_ROCKCHIP=y
 CONFIG_PHY_ROCKCHIP_USB=y
 CONFIG_PHY_ROCKCHIP_INNO_USB2=y
 CONFIG_PHY_ROCKCHIP_INNO_USB3=y
-CONFIG_PHY_ROCKCHIP_EMMC=y
 CONFIG_PHY_ROCKCHIP_INNO_MIPI_DPHY=y
 CONFIG_PHY_ROCKCHIP_PCIE=y
 CONFIG_ANDROID=y
-- 
2.35.3

